File: src/main/java/com/bookmakerApp/view/BookmakerClientApp/BookmakerClientApp/Data/Model/Payment/PaymentDto.cs
using System;

namespace BookmakerClientApp.Data.Model.Payment
{
	public class PaymentDto
	{
		private string statusCode;
		private string redirectUri;
		private string orderId;
		private string currencyCode;
		private Decimal totalAmount;
		private int numberOfPages;

		public string StatusCode { get => statusCode; set => statusCode = value; }
		public string RedirectUri { get => redirectUri; set => redirectUri = value; }
		public string OrderId { get => orderId; set => orderId = value; }
		public string CurrencyCode { get => currencyCode; set => currencyCode = value; }
		public decimal TotalAmount { get => totalAmount; set => totalAmount = value; }
		public int NumberOfPages { get => numberOfPages; set => numberOfPages = value; }
	}
}

File: src/main/java/com/bookmakerApp/facade/dtos/payment/PaymentDto.java
package com.bookmakerApp.facade.dtos.payment;

import lombok.Builder;
import lombok.Getter;

import java.math.BigDecimal;

@Getter
@Builder
public class PaymentDto {
    private String statusCode;
    private String redirectUri;
    private String orderId;
    private String currencyCode;
    private BigDecimal totalAmount;
    private int numberOfPages;
}
